am/scripted-decision/src/get-location-from-address.js

```diff
@@ -0,0 +1,166 @@
+/**
+ * @file Determine the authenticating user location based on their address
```

Comment:
I would say "IP address" instead of just "address"

Reply:
Oh!!!! I just realize that you are literally using the postal address, not the user's IP address! I recommend changing the script to use the IP address, so that there are no prerequisits in user data. And maybe instead of extracting only longitude and latitude also resolve to the country for country-based geo-fencing? I just find longitude and latitude not very demoable without a map.

Reply:
@vscheuber, I added a `get-location-from-ip-address.js` example, and updated the `get-location-from-postal-address.js` one. The idea is that they can obtain location information independently, in separate nodes. Then both locations could be saved, compared, etc. Not sure whether either of them is useful enough separately, or it is worth to combine them in a single script/extend (say, for a comparison example, or to add fraud check for the IP one, etc.).